src/reactions/MessageReactionList.js

```diff
@@ -3,6 +3,7 @@ import React, { PureComponent } from 'react';
 import { View } from 'react-native';
 import { createMaterialTopTabNavigator } from 'react-navigation';
 import type { NavigationScreenProp } from 'react-navigation';
+import * as logging from '../utils/logging';
```

Comment:
nit: we group third-party imports (more precisely, out-of-tree -- starting with an NPM package name) first, then a blank line before local imports (with a leading `./` or `../`). So this should go in the second group of imports.

I've actually been half-tempted to abandon that practice, until/unless we get a tool involved which will maintain it automatically (and probably also keep the imports sorted, which would be a nice small improvement but I've considered definitely too annoying to maintain by hand.) But if we do we should remove all the existing blank-line separators of this kind, so as not to have a mix of conventions. For the moment, easy to fix any given instance like this one.

Reply:
Ah, thanks for catching that. Fixed in https://github.com/zulip/zulip-mobile/commit/4c3bc34e14c5ebdcee40cb587a82eab3e2ecdfd3.